oci/private/image.sh

```diff
@@ -174,6 +180,13 @@ for ARG in "$@"; do
       jq --rawfile annotations "${ARG#--annotations=}" \
       '.annotations += ([($annotations | split("\n") | .[] | select(. != ""))] | map(. | split("=")) | map({key: .[0], value: .[1:] | join("=")}) | from_entries)' |
       update_manifest
+
+    created_date="$(jq -r -n \
+      --arg created_key "${OPENCONTAINERS_CREATED_KEY}" \
+      --arg default_created_date "${DEFAULT_CREATED_DATE}" \
+      --rawfile annotations "${ARG#--annotations=}" \
+      '([$annotations | split("\n") | .[] | select(. != "") | split("=") | {key: .[0], value: .[1]}]) | map(select(.key==$created_key)) | first | .value // $default_created_date')"
```

Comment:
I'm sure we could remove some duplication in this and the above annotations handling